lstmcpipe/config/paths_config.py

```diff
@@ -267,13 +269,13 @@ def models_path(self):
 
     @property
     def train_pipe(self):
-        paths = [{
+        paths = {
```

Comment:
I would suggest to return to list also for the train_pipe stage, despite it contains a single item in the list. 
We won't need to change the test in were we check the type of each class attribute ... (see other comment)

Reply:
I only tried to match your implementation of the stages here 😅 
As I understood the current implementation, we could not run several trainings in parallel anyway (because of the jobs dependency handling).
I would be happy if that can be done though.